satpy/tests/enhancement_tests/test_enhancements.py

```diff
@@ -32,7 +32,7 @@
 # - tmp_path
 
 
-def run_and_check_enhancement(func, data, expected, **kwargs):
+def run_and_check_enhancement(func, data, expected, match_dtype=False, **kwargs):
```

Comment:
Can we make another function instead of passing a boolean here? eg `run_and_check_enhancement_with_dtype`
I think it will be cleaner.

Reply:
I'll see how well the test can be chopped to pieces.

Reply:
Adjusted in https://github.com/pytroll/satpy/pull/2956/commits/516992329e6c880c8bfb226f41d5d53c6e5476d4